xcube/core/gen/gen.py

```diff
@@ -371,6 +373,9 @@ def step8(input_slice):
 def _update_cube_attrs(output_writer: DatasetIO, output_path: str,
```

Comment:
Please rename this function from `_update_cube_attrs` into `_update_cube` as it no longer only updates attributes.